src/page-modules/contact/utils.ts

```diff
@@ -1,3 +1,20 @@
+import { ContactForm } from './components/form-selector';
+
 export const shouldShowContactPage = (): boolean => {
   return process.env.NEXT_PUBLIC_CONTACT_API_URL ? true : false;
 };
+
+export const renderSelectedPage = (
```

Comment:
See comment above, but I believe this should be handled by the next js page router